airbyte-integrations/connectors/source-mssql/src/main/java/io/airbyte/integrations/source/mssql/MssqlCdcStateHandler.java

```diff
@@ -56,4 +58,9 @@ public AirbyteMessage saveStateAfterCompletionOfSnapshotOfNewStreams() {
     throw new RuntimeException("Snapshot of individual tables is not implemented in MSSQL");
   }
 
+  @Override
+  public boolean compressSchemaHistoryForState() {
```

Comment:
Should we just flip this to true and refactor rest of code? Schema history compression currently applies to MySQL & MsSQL, so could be a way to simplify code + make sure we apply it to default to any future connector that has a notion of schema history

Reply:
I agree in principle but I'd like to do that change in a separate PR, if that's OK.

Reply:
That's fine, followup works